addons/sale_stock/report/report_stock_forecasted.py

```diff
@@ -31,3 +32,17 @@ def _product_sale_domain(self, product_template_ids, product_variant_ids):
         if warehouse_id:
             domain += [('warehouse_id', '=', warehouse_id)]
         return domain
+
+    def _get_order_name(self):
```

Comment:
Can be avoided (and should be) if you use the two values context passing (related to the comment on the order_name).